cnf-certification-test/identifiers/identifiers.go

```diff
@@ -227,16 +227,15 @@ func InitCatalog() map[claim.Identifier]claim.TestCaseDescription {
 		`Checks that pods do not use projected volumes and service account tokens`,
 		ProjectedVolumeServiceAccountRemediation,
 		InformativeResult,
-		NoDocumentedProcess,
+		ElaborateOnWhyItIsNeeded,
 		bestPracticeDocV1dot4URL+" Section 4.6.24",
 		true,
 		map[string]string{
-			FarEdge:  Optional,
-			Telco:    Optional,
+			FarEdge:  Mandatory,
+			Telco:    Mandatory,
 			NonTelco: Optional,
-			Extended: Mandatory,
-		},
-		TagExtended)
+			Extended: Mandatory},
+		TagCommon, TagTelco)
```

Comment:
As per our last discussion regarding tc labels, shouldn't this be only "TagTelco" ?